projects/ngx-powerbi/src/lib/ngx-powerbi.component.ts

```diff
@@ -41,7 +41,7 @@ export class NgxPowerBiComponent implements OnInit, OnChanges, OnDestroy {
   @Input() name: string;
   @Input() options: models.ISettings;
   @Output() embedded = new EventEmitter<number>();
-  @ViewChild('ngxPowerBiIFrame') ngxPowerBiIFrameRef: ElementRef;
+  @ViewChild('ngxPowerBiIFrame', {static: true}) ngxPowerBiIFrameRef: ElementRef;
```

Comment:
I hope you have tested the OnInit -> AfterViewInit and this {static: true} change?

Reply:
I was already using ngx-powerbi in my application and I had to do this change to make it work with Angular 9.